.gitlab-ci.yml

```diff
@@ -15,7 +15,9 @@ stages:
 
 # Template for jobs that want to build everything from source (much slower)
 .job_template: &src_job_template
-  image: osrf/ros2:nightly
+  image:
```

Comment:
I moved this up to the template so all the jobs get `entrypoint: [""]` automatically.  Previously it was only in the build_isolated job